include/fastdds/dds/subscriber/DataReader.hpp

```diff
@@ -289,6 +289,35 @@ class DataReader : public DomainEntity
             ViewStateMask view_states = ANY_VIEW_STATE,
             InstanceStateMask instance_states = ANY_INSTANCE_STATE);
 
+    /** NOT YET IMPLEMENTED
+     * This operation accesses via ‘read’ the samples that match the criteria specified in the @ref ReadCondition.
+     * This operation is especially useful in combination with @ref QueryCondition to filter data samples based on the
+     * content.
+     *
+     * The specified @ref ReadCondition must be attached to the DataReader; otherwise the operation will fail and return
+     * RETCODE_PRECONDITION_NOT_MET.
+     *
+     * In case the @ref ReadCondition is a ‘plain’ @ref ReadCondition and not the specialized @ref QueryCondition, the
+     * operation is equivalent to calling read and passing as @c sample_states, @c view_states and @c instance_states
+     * the value of the corresponding attributes in @c a_condition. Using this operation the application can avoid
+     * repeating the same parameters specified when creating the ReadCondition.
+     *
+     * The samples are accessed with the same semantics as the read operation. If the DataReader has no samples that
+     * meet the constraints, the return value will be RETCODE_NO_DATA.
+     *
+     * @param [in,out] data_values     A LoanableCollection object where the received data samples will be returned.
```

Comment:
**Change along the file:** `@param[in|out|in,out]` do not have a space between `param` and `[`. 
https://www.doxygen.nl/manual/commands.html#cmdparam